pkg/apis/config/v1alpha1/resourceinterpretercustomization_types.go

```diff
@@ -241,8 +287,14 @@ type DependencyInterpretation struct {
 	//             return dependencies
 	//         end
 	//
-	// LuaScript only holds the function body part, take the `desiredObj` as the function
-	// parameters or global variables, and finally returns the dependent resources.
+	// The content of the LuaScript needs to be a whole function including both
```

Comment:
I think it shall be ` dependencies[1] = ...`

https://github.com/karmada-io/karmada/blob/e7b9a11fbd934be2a75b449e67f634ef4052a68e/pkg/apis/config/v1alpha1/resourceinterpretercustomization_types.go#L284-L285

Reply:
I agree, but it also makes the example hard to read.